runtime/interpreter/decode.go

```diff
@@ -17,21 +18,22 @@ import (
 // A Decoder decodes CBOR-encoded representations of values.
 //
 type Decoder struct {
-	dec *cbor.Decoder
+	decoder *cbor.Decoder
+	owner   *common.Address
 }
 
 // Decode returns a value decoded from its CBOR-encoded representation,
 // for the given owner (can be `nil`).
```

Comment:
It would be good to update this comment to describe the `path` parameter.

Reply:
Documented it for both for the encoding and the decoding function in ec91981bb46499cc7c3ffafeef5596a7837295d2